packages/assets-query-client/src/al-assets-query-client.ts

```diff
@@ -223,18 +223,20 @@ export class AlAssetsQueryClientInstance {
   /**
    * Dispose Remediations
    * PUT
-   * /remediations/v1/:account_id/deployments/:deployment_id/remediations
-   * "https://api.cloudinsight.alertlogic.com/remediations/v1/10000001/deployments/347203EF-134C-1005-8499-1289DB15AB31/remediations"
+   * /assets_query/v2/:account_id/remediations
+   * "https://api.cloudinsight.alertlogic.com/assets_query/v2/10000001/remediations"
    * -d '{"operation": "dispose_remediations", "reason": "Acceptable Risk",
    * "comment": "This risk is acceptable", "expires": 0,
-   * "remediation_items": ["/al/15000001:814C2911-09BB-1005-9916-7831C1BAC182/remediation-item/0536575B914C32C8A5D28415D02E4545"]}
+   * "remediation_ids": ["ids_upgrade_hardware"], "deployment_ids":["0A2DC25F-5B5A-4A93-9413-1A8D6F87489E"], filters:[]}'
    */
-  async disposeRemediations(accountId: string, deploymentId: string, remediationData: {operation: string, reason: string, comment: string, expires: number, remediation_items: string[]}) {
+  async disposeRemediations(accountId: string, remediationData: { deployment_ids: string[], operation: string, filters: string[], remediation_ids: string[],
+                                                                  reason: string, comment: string, expires: number}) {
     return this.client.put<any>({
-      account_id: accountId,
-      service_name: 'remediations',
-      path: `/deployments/${deploymentId}/remediations`,
-      data: remediationData,
+        account_id: accountId,
+        service_name: 'assets_query',
+        path: 'remediations',
+        version: 'v2',
+        data: remediationData,
```

Comment:
Looking at the API docs - https://console.account.alertlogic.com/users/api/assets_query/#api-Remediations-DisposeRemediations

In the remediationData param, needing to make deployment_ids prop optional, also there should be a `vulnerability_ids` property present/

API docs also state that the comment must be a value of `dispose_remediations`, I would therefore not have the operation property as something that can be supplied and instead hard code this into the method.

So something like:

`let baseRemediationData = { operation: 'dispose_remediations' }`

and then merge in the props from the incoming `remediationData`